dvc/remote/base.py

```diff
@@ -450,6 +452,7 @@ def create_taskset(amount):
 
     @index_locked
     def push(self, cache, named_cache, jobs=None, show_checksums=False):
+        self.odb.migrate_config()
```

Comment:
Updating/writing the config should really be done as needed after an initial `odb.write/save/etc` operation, but since we don't do anything else with ODB at the moment writing the config file to a remote on `push` should be sufficient for 2.0